client/extensions/woocommerce/app/settings/shipping/shipping-zone/shipping-zone-location-list.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import ExtendedHeader from 'woocommerce/components/extended-header';
+import List from 'woocommerce/components/list/list';
+import ListItem from 'woocommerce/components/list/list-item';
+import ListHeader from 'woocommerce/components/list/list-header';
+import ListItemField from 'woocommerce/components/list/list-item-field';
+import LocationFlag from 'woocommerce/components/location-flag';
+import ShippingZoneLocationDialog from './shipping-zone-location-dialog';
+import { decodeEntities } from 'lib/formatting';
+import { bindActionCreatorsWithSiteId } from 'woocommerce/lib/redux-utils';
+import { getCurrentlyEditingShippingZoneLocationsList } from 'woocommerce/state/ui/shipping/zones/locations/selectors';
+import { openEditLocations } from 'woocommerce/state/ui/shipping/zones/locations/actions';
+
+const ShippingZoneLocationList = ( { siteId, loaded, translate, locations, actions, onChange } ) => {
+	const getLocationDescription = ( location ) => {
+		switch ( location.type ) {
+			case 'continent':
+				return translate( 'All countries' );
+			case 'country':
+				if ( location.postcodeFilter ) {
+					return translate( 'Specific postcodes: %s', { args: [ location.postcodeFilter ] } );
+				}
+
+				return translate( 'Whole country' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **69** times:
`translate( 'Country' )` *ES Score: 7*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/14275-shipping-locations-ui)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).